packages/block-editor/src/components/rich-text/README.md

```diff
@@ -14,7 +14,7 @@ Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs
 
 ### `tagName: String`
 
-*Default: `div`.* The [tag name](https://www.w3.org/TR/html51/syntax.html#tag-name) of the editable element. Elements that display inline are not supported.
```

Comment:
This is still correct. If you _style_ any element inline, it will have problems in some browsers and we'll log a warning in the console.